Remove bad examples for PR templates

leoheitmannruiz opened the pull request:
This PR removes bad examples for what to put in a PR template. The current examples seem to be pasted with little thought: why would you want to add a "description of the changes proposed in the pull request" in a PR template?

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

leoheitmannruiz committed:
Remove bad examples for PR templates

nguyenalex836 commented:
@leoheitmannruiz Thanks for opening this PR! 👋 Out of curiosity - are there alternative examples you would replace these with, that someone who is newer to creating PR templates might be able to reference?

leoheitmannruiz committed:
Update phrasing

nguyenalex836 committed:
Update creating-a-pull-request-template-for-your-repository.md

"This template could consist of asking to include:" 
- Added the word template
- Fixed a typo in the word "asking"

nguyenalex836 commented:
@leoheitmannruiz Awesome, thank you! 💛 I made a tiny adjustment to line 36 (fixed a typo in the word "asking", added the word "template')

I'll update the branch and get this merged once tests are passing 🍏

nguyenalex836 committed:
Merge branch 'main' into patch-1
